skimage/filters/tests/test_gaussian.py

```diff
@@ -37,13 +37,13 @@ def test_multichannel():
                         [gaussian_rgb_a[..., i].mean() for i in range(3)])
     # Test multichannel = None
     with expected_warnings(['multichannel']):
-        gaussian_rgb_a = gaussian_filter(a, sigma=1, mode='reflect')
+        gaussian_rgb_a = gaussian(a, sigma=1, mode='reflect')
     # Check that the mean value is conserved in each channel
     # (color channels are not mixed together)
     assert np.allclose([a[..., i].mean() for i in range(3)],
                         [gaussian_rgb_a[..., i].mean() for i in range(3)])
     # Iterable sigma
-    gaussian_rgb_a = gaussian_filter(a, sigma=[1, 2], mode='reflect',
+    gaussian_rgb_a = gaussian(a, sigma=[1, 2], mode='reflect',
                                      multichannel=True)
```

Comment:
It's a PEP8 style point about indentation, see https://www.python.org/dev/peps/pep-0008/#indentation
Here the second line or arguments should be aligned with the first one, so ``multichannel=True`` should be just under `a`.